plugins/Scheduler/test.py

```diff
@@ -74,6 +74,13 @@ def test00RemoveZero(self):
         timeFastForward(5)
         self.assertNoResponse(' ', timeout=1)
 
+    def testRemind(self):
+        self.assertNotError('scheduler remind 5 testRemind')
+        timeFastForward(5)
+        self.assertResponse(' ', 'Reminder: testRemind')
```

Comment:
This doesn't make sure the reminder isn't sent too early.

Instead you should use `timeFastForward` on a shorter duration, use `assertNoResponse` to check the reminder isn't set yet, `timeFastForward` again, then `assertResponse`.